src/web/mjs/connectors/Piccoma.mjs

```diff
@@ -0,0 +1,275 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Piccoma extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'piccoma';
+        super.label = 'Piccoma';
+        this.tags = ['webtoon', 'japanese'];
+        this.url = 'https://piccoma.com/web';
+    }
+
+    async _getMangaFromURI(uri) {
+        const id = uri.pathname.split('/')[3];
+        const request = new Request(uri.href);
+        const [data] = await this.fetchDOM(request, '.PCM-productTitle');
+        const title = data.textContent.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const mangas = [];
+        let totalPage = 1;
+        for (let i = 1; i <= totalPage; i++) {
+            const request = new Request(`${this.url}/next_page/list?result_id=2&list_type=C&sort_type=N&page_id=${i}`, this.requestOptions);
+            const res = await this.fetchJSON(request);
+            totalPage = res.data.total_page;
+            const products = res.data.products;
+            mangas.push(...products.map(({ id, title }) => {
+                return { id, title };
+            }));
+        }
+        return mangas;
+    }
+
+    async _getChapters(manga) {
+        const request = new Request(`${this.url}/product/${manga.id}/episodes?etype=E`);
+        const data = await this.fetchDOM(request, '.PCM-product_episodeList > a');
+        return data.map(element => {
+            return {
+                id: `${manga.id}/${element.dataset.episode_id}`,
+                title: element.querySelector('.PCM-epList_title').textContent.trim(),
+            };
+        });
+    }
+
+    async _getPages(chapter) {
+        const request = new Request(`${this.url}/viewer/${chapter.id}`);
+        const _pdata = await Engine.Request.fetchUI(request, 'window._pdata_ || {}');
+        const { img, ...pdata } = _pdata;
+        if (img == null) {
+            throw new Error(`The chapter '${chapter.title}' is neither public, nor purchased!`);
+        }
+        return img
```

Comment:
Either rename the local variable `img` to e.g. `images` or use a different name within the arrow functions in filter/map to improve readability (distinguishable variable names `img`)